ipywidgets/widgets/widget.py

```diff
@@ -223,6 +272,10 @@ def open(self):
             if buffers:
                 # FIXME: workaround ipykernel missing binary message support in open-on-init
                 # send state with binary elements as second message
+                # TODO: if this gets fixed, _split_state_buffers does not need to have a seperate
```

Comment:
It is less trivial then I thought, this means that the buffers will need to finally finds it's way [here](https://github.com/ipython/ipywidgets/blob/master/jupyter-js-widgets/src/manager-base.ts#L284) right?